microfrontends/SignDocuments/package.json

```diff
@@ -32,6 +32,7 @@
     "react-scrollbars-custom": "^4.1.1",
     "react-signature-canvas": "^1.0.6",
     "reactour": "^1.19.1",
+    "validator": "^13.11.0",
```

Comment:
It looks like a large library.
<img width="412" alt="image" src="https://github.com/OpenSignLabs/OpenSign/assets/5486116/454f54a3-75f5-437f-8c82-19fd186a834c">
Can we consider handling this issue without validator package?

Reply:
Yes we can do it.

Reply:
Hello @nxglabs it is taken care.